filters-api.md

```diff
@@ -19,38 +23,80 @@ The supported filter fields as of v1.96 are:
 * `label`
 * `annotation` (same syntax as label, see examples)
 
-The supported filter ops as of v1.96 are:
-
-* `:` (equality, or "contains" if an array type)
-* `!:` (inequality, or "not contains" if an array type)
-
-Filter values are strings surrounded by `"`. Multiple values can be separated by commas `,`, representing logical OR.
-
-Each individual filter is separated by a `+`, representing logical AND.
+Added in v1.105 of Kubecost:
+* `department`
+* `environment`
+* `owner`
+* `product`
+* `team`
+
+The supported filter ops in v1.96 of Kubecost are:
```

Comment:
suggest replacing `ops` with `operators`
```suggestion
The supported filter operators in v1.96 of Kubecost are:
```

Reply:
Great suggestion, thank you